src/btree/bt_curprev.c

```diff
@@ -753,6 +753,11 @@ __wt_btcur_prev(WT_CURSOR_BTREE *cbt, bool truncating)
 
     WT_STAT_CONN_DSRC_INCR(session, cursor_prev);
 
+    /* Track prev calls during HS wrapup */
+    if (F_ISSET(session, WT_SESSION_HS_WRAPUP)) {
+        session->reconcile_stats.hs_wrapup_next_prev_calls++;
```

Comment:
We do not need {} for a single line if statement. Remove it in the other location also.